object_detection.py

```diff
@@ -20,8 +20,9 @@
 # Imports the Anki Vector SDK
 import anki_vector
 from anki_vector.util import degrees
+import anki_vector.camera
 
-robot = anki_vector.Robot(anki_vector.util.parse_command_args().serial, enable_camera_feed=True)
+robot = anki_vector.Robot(anki_vector.util.parse_command_args().serial)#, enable_camera_feed=True)
```

Comment:
` enable_camera_feed=True` defualt is `True` in the latest SDK?

Reply:
It seems to work with that line commented out. it was giving the error listed in the bugs section when it was run with it in there with the new SDK. I tried it out, it works.

Reply:
Can you see the video streaming shows on Vector's screen?

Reply:
robot = anki_vector.Robot(anki_vector.util.parse_command_args().serial, enable_camera_feed=True)
TypeError: __init__() got an unexpected keyword argument 'enable_camera_feed'